forte/data/data_store.py

```diff
@@ -939,6 +943,49 @@ def _parse_onto_file(self):
             return
         raise NotImplementedError
 
+    @staticmethod
+    def get_class(type_name):
+        r"""Returns the ontology class based on class name. Only a class that exists in current
+        DataStore will be returned.
+
+        Args:
+            type_name (str): Fully qualified name of the class.
+        Returns:
+            The target class.
+
+        Raises:
+            ValueError: If class is not found based on :attr:`class_name` in DataStore.
+        """
+        if type_name not in DataStore._type_attributes:
+            raise ValueError(f"Class not found in DataStore: {type_name}")
+        return get_class(type_name)
+
+    @staticmethod
+    def get_subtypes(type_name):
+        r"""Returns a complete list of subclasses based on class name.
```

Comment:
Actually not a complete list, only the subclasses that are "known" to us. You should explain here which types are "known".